python/packages/ai/teams/ai/planner/openai_planner_options.py

```diff
@@ -11,30 +11,9 @@
 class OpenAIPlannerOptions:
     api_key: str
     default_model: str
-    prompt_folder: str
+    prompt_folder: str = "prompts"
```

Comment:
Also why we need to have default prompt_folder?

Reply:
see above comment.

Reply:
Please also consider making it to be a keyword only parameter.